configure.ac

```diff
@@ -133,6 +133,14 @@ ODP_CHECK_CFLAG([-Wformat-overflow=0])
 # header structures). Generate only warnings on those, not errors.
 ODP_CHECK_CFLAG([-Wno-error=address-of-packed-member])
 
+# GCC 10 sometimes gets confused about oject sizes and gives bogus warnings.
+# Make the affected warnings generate only warnings, not errors.
+case `($CC --version) 2>/dev/null` in
+     'gcc (GCC) 10'*)
```

Comment:
This check doesn't work on Ubuntu 20.04.

```
$ gcc-10 --version
gcc-10 (Ubuntu 10-20200411-0ubuntu1) 10.0.1 20200411 (experimental) [master revision bb87d5cc77d:75961caccb7:f883c46b4877f637e0fa5025b4d6b5c9040ec566]
```

Reply:
Fixed in v2.